intro-to-oop-relationships/problem-set-oop-relationships.md

```diff
@@ -204,6 +204,7 @@ print(bear.make_noise())
 ##### !end-options
 
 ##### !answer
+*
```

Comment:
This was intentional! There was a bug where it wouldn't publish on Learn because the answer was missing a `*`. Note to future selves: answers for checkbox questions need a `*`.